src/scripts/spotify-playlist-metadata-to-id.ts

```diff
@@ -0,0 +1,92 @@
+import dbContext from "../database_context";
+import type { PlaylistMetadata } from "../interfaces/playlist_metadata";
+
+(async () => {
+    if (require.main === module) {
+        {
+            const metadataList = await dbContext.kmq
+                .selectFrom("game_options")
+                .selectAll()
+                .where("option_name", "=", "spotifyPlaylistMetadata")
+                .where("option_value", "<>", "null")
+                .execute();
+
+            const playlistIDs: Array<{
+                guild_id: string;
+                option_name: string;
+                option_value: string;
+                client_id: string;
+            }> = [];
+
+            for (const metadata of metadataList) {
+                if (metadata.option_value) {
+                    const optionValue: PlaylistMetadata = JSON.parse(
+                        metadata.option_value
+                    );
+
+                    const playlistID = optionValue.playlistID;
+                    playlistIDs.push({
+                        guild_id: metadata.guild_id,
+                        option_name: "spotifyPlaylistID",
+                        option_value: playlistID,
+                        client_id: metadata.client_id,
+                    });
+                }
+            }
+
+            await dbContext.kmq
+                .insertInto("game_options")
+                .values(playlistIDs)
+                .execute();
+
+            await dbContext.kmq
+                .deleteFrom("game_options")
+                .where("option_name", "=", "spotifyPlaylistMetadata")
+                .execute();
+        }
+
+        {
```

Comment:
Is this codeblock duplicated?

Reply:
One is for options, the other is for presets. Putting in a for loop with those 2 doesn't work well since there are fields that exist in one but not the other